.gitignore

```diff
@@ -5,6 +5,7 @@ work
 # Binaries and downloaded resources
 src/main/resources/matlab-script-generator.zip
 src/main/resources/run_matlab_command.*
+src/main/resources/**/run-matlab-command*
```

Comment:
Why do we need this ? are you adding anything under resource folder which needs to be ignored ?

Reply:
I think instead of addition, it needs to be a fix for line 7. It is there to ignore the `run-matlab-command` executables which are downloaded when you install the plugin.
@tsharmaMW can you update it accordingly